src/config/Config.ts

```diff
@@ -155,6 +155,12 @@ export interface ClientConfig {
      */
     properties?: Properties;
 
+    /**
+     * Metrics config. With this config, you enable collecting the client metrics and sending them to the cluster.
+     * After enabling you can monitor the clients that are connected to your Hazelcast cluster,
```

Comment:
How about: "You can monitor connected clients using Hazelcast Management Center once the metrics collection is enabled"?